code/script.js

```diff
@@ -64,33 +64,39 @@ do {
     4 - Caesar Salad`);
       subtypeChoiceFood = ["Garden Salad", "Greek Salad", "Caprese Salad", "Caesar Salad"];
       break;
-    default: //i don't think I need this? If an invalid option is chosen, show an error message and exit the program. //alert("Invalid choice."); //process.exit(1);
+    default:
+      alert('Your choice is not available, Please try again');
   }
 }
-while (subtypeChoiceNum >= 5); //do...while loop if user does not enter a valid option (1,2 or 3)
+while (subtypeChoiceNum < 1 || subtypeChoiceNum > 4); //do...while loop if user does not enter a valid option (1,2, 3 or 4) they are asked again. It does not work if user inputs text. Diego has helped with tips and advice on how to check if a variable is a number.
 
 alert(`Great Choice, You chose Number ${subtypeChoiceNum} - ${subtypeChoiceFood[subtypeChoiceNum - 1]}`);
 
```

Comment:
I do find the variable names throughout the code a little too similar in name, though they are logical it might be easier to differentiate between them if they have less similar names?